solutions/5.js

```diff
@@ -29,6 +29,25 @@ const mergeTogether = (firstHalf, secondHalf) => {
   return sortedResult;
 };
 
+// Daniel Soper
+// sort an array of strings or numbers in ascending order
+const solution2  = (arr) => {
+  let max;
+  if (arr.length === 1){
+    return arr; 
+  }
+  for (i=0; i < arr.length; i++) {
+    for(j=i+1; j< arr.length; j++) {
+      if (arr[i] > arr[j]) {
+        temp = arr[i];
+        arr[i] = arr[j];
+        arr[j] = temp;
+      }
+    } 
+  } 
+  return arr 
+}
```

Comment:
@drsoper please be consistent with semicolons. you are using them in the return at line 37 but not at 48.